frame/support/procedural/src/storage/instance_trait.rs

```diff
@@ -24,7 +24,6 @@ use super::DeclStorageDefExt;
 
 const NUMBER_OF_INSTANCE: usize = 16;
 pub(crate) const INHERENT_INSTANCE_NAME: &str = "__InherentHiddenInstance";
-pub(crate) const DEFAULT_INSTANTIABLE_TRAIT_NAME: &str = "__GeneratedInstantiable";
```

Comment:
note: __GeneratedInstantiable is no longer generated because it was supposed to be only internally used (and so all our internal use now use frame_support::traits::Instance directly)